spec/actors/create_with_files_actor_spec.rb

```diff
@@ -22,7 +22,7 @@
       context "when uploaded_file_ids include nil" do
         let(:uploaded_file_ids) { [nil, uploaded_file1.id, nil] }
         it "will discard those nil values when attempting to find the associated UploadedFile" do
-          allow(AttachFilesToWorkJob).to receive(:perform_later)
+          expect(AttachFilesToWorkJob).to receive(:perform_later)
```

Comment:
I don't think the idea was to test for this here, meerly to prevent it from occurring.